README.md

```diff
@@ -1,39 +1,108 @@
 # Amazon Textract Document Understanding Solution
 
+## CICD Deploy
+
+### Requirements
+
+- aws cli
+
+### Getting Started with Full Deploy
+
+- Create a bucket to act as the target Amazon S3 distribution bucket
+
+_Note:_ You will have to create an S3 bucket with the template 'my-bucket-name-<aws_region>'; aws_region is where you are testing the customized solution. Also, the assets in this bucket should be publicly accessible.
+
+For example, you create a bucket called `document-understanding-bucket-us-east-1`,
+
+- Now build the distributable:
+
+```
+chmod +x ./deployment/build-s3-dist.sh
```

Comment:
Could you double check this command?
I cannot find the build-s3-dist.sh script

Reply:
File has been added to PR